engine/src/w32-clipboard.cpp

```diff
@@ -312,7 +313,10 @@ MCStringRef MCWin32RawClipboardCommon::DecodeTransferredFileList(MCDataRef p_dat
 	}
 
 	// Done
-	return t_decoded;
+    MCAutoStringRef t_path;
+    MCS_pathfromnative(t_decoded, &t_path);
+    MCValueRelease(t_decoded);
+    return *t_path;
 }
```

Comment:
This should return t_path.Take() otherwise the MCStringRef will be auto-decremented and the memory released